### PR TITLE
Avoid tag duplicates

### LINKED ISSUE #261: Remove duplicate tags on save
Awesomplete lets (very inattentive) users to enter more than once a tag...

### PR DESCRIPTION
**Pull request by ArthurHoaro**
* Prevent duplicate client side with awesomplete
 * Prevent duplicate server side (save_edit processing)

Fixes #261

### CODE REVIEW

**Comment by alexisju** [comment]
Tested (both side) : works great :-)

File: tpl/editlink.html [hunk 1]
```diff
@@ -52,6 +52,21 @@
         },
         minChars: 1
     });
+
+    var input = document.querySelector('#lf_tags');
```
Comment (virtualtam) [inline tpl/editlink.html]: could you add a brief comment to explain that the list of suggested tags is being pruned to prevent duplication?

**Comment by virtualtam** [comment]
Works fine for both JS & PHP

*TRAVIS, WHY U NO RUN TESTS?*

**Comment by ArthurHoaro** [comment]
> TRAVIS, WHY U NO RUN TESTS?

No idea, but [this works](http://stackoverflow.com/a/25046839/1484919).

**Comment by virtualtam** [comment]
+1
could you add a TODO or FIXME comment to remember removing the JS workaround when [awesomeplete/#16749](https://github.com/LeaVerou/awesomplete/issues/16749) has been solved?

**Comment by ArthurHoaro** [comment]
Done.

**Pull request merged**